### ISSUE TITLE
Quantile Regression

### ISSUE THREAD

**Issue by s3alfisc** [py-econometrics/pyfixest#667]
Implement the fast exact algorithm described in [here](https://boris.unibe.ch/150869/8/revision03.pdf).

**Comment by vincentarelbundock** [comment]
IIUC, this is already implemented, so the issue can probably be closed.

**Comment by s3alfisc** [comment]
Yes, thanks for the pointer @vincentarelbundock! Sometimes I forget to clean up around here... 😅
